compiler/rustc_error_codes/src/error_codes/E0699.md

```diff
@@ -1,3 +1,5 @@
+#### Note: this error code is no longer emitted by the compiler.
+
```

Comment:
You also need to remove `E0699` from the failing code example attributes just below.